React InstantSearch/injected-content/src/App.css

```
@@ -132,6 +132,38 @@ em {
   display: block;
 }
 
+.Recommend .ais-Hits-item:nth-of-type(5) {
+  color: #00aeff;
+  border-color: currentColor;
+}
+
+.Recommend .ais-Hits-injected .FrequentlyBoughtTogether {
+  color: #00aeff;
+  background-color: #e5f1f7;
+  border-color: currentColor;
+  width: 100%;
+  box-shadow: none;
+}
+
+.Recommend .ais-Hits-injected .FrequentlyBoughtTogether h2 {
+  margin-top: 0;
+}
+
+.Recommend .ais-Hits-injected .FrequentlyBoughtTogether ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  flex-grow: 1;
+  display: grid;
+  grid-auto-flow: column;
```

This way, each colum takes exactly the same space even with longer text.

```suggestion
  grid-auto-flow: column;
  grid-auto-columns: 1fr;
```